src/datumaro/plugins/data_formats/yolo/base.py

```diff
@@ -316,6 +316,9 @@ def __init__(
         if not osp.isdir(config_path):
             raise DatasetImportError(f"{config_path} should be a directory.")
 
+        if not urls:
+            raise DatasetImportError(f"urls should be specified for {self.__class__.__name__}.")
```

Comment:
Could it be more descriptive such as
```suggestion
            raise DatasetImportError(
                f"`urls` should be specified for {self.__class__.__name__}, "
                f"if you want to import a dataset with using this {self.__class__.__name__} directly. "
                "In most case, it happens by giving an incorrect format name to the import interface. "
                'Please consider to import your dataset with this format name "yolo" such as `Dataset.import_from(..., format="yolo")`'
            )
```